cli/flags.rs

```diff
@@ -335,12 +335,82 @@ This command has implicit access to all permissions (equivalent to deno run --al
 Automatically downloads Prettier dependencies on first run.
 
   deno fmt myfile1.ts myfile2.ts",
-        ).arg(
+        )
+        .arg(
           Arg::with_name("stdout")
             .long("stdout")
             .help("Output formated code to stdout")
             .takes_value(false),
-        ).arg(
+        )
+        .arg(
+          Arg::with_name("prettier-print-width")
```

Comment:
/cc @ry ready for review now.
I am not sure if this is a best practice.

Currently using the flag format `--prettier-print-width=80`

Is it better to use `--prettier-flags=--print-width=80` like `--v8-flags`?

Reply:
I think the way you did it is better. But can you use —fmt-print-width= rather ? It’s shorter and matches the sub command